share/spice/github_jobs/github_jobs.js

```diff
@@ -16,24 +18,40 @@
 
         var sourceUrl = 'https://jobs.github.com/positions?description=' + encodeURIComponent(jobs['description']) + "&location=" + encodeURIComponent(jobs['location']);
 
-        Spice.add({
-            id: 'github_jobs',
-            name: 'Jobs',
-            data: api_result,
-            meta: {
-                sourceUrl: sourceUrl,
-                sourceName: 'GitHub',
-                sourceIcon: true,
-                itemType: 'Jobs'
-            },
-            templates: {
-                group: 'base',
-                options: {
-                    content: Spice.github_jobs.content
+        DDG.require('moment.js', function() {
+            Spice.add({
+                id: 'github_jobs',
+                name: 'Jobs',
+                data: api_result,
+                meta: {
+                    sourceUrl: sourceUrl,
+                    sourceName: 'GitHub',
+                    sourceIcon: true,
+                    itemType: 'Jobs'
+                },
+                normalize: function(item) {
+                    return {
+                        title: item.title,
+                        subtitle: item.company,
+                        location: item.location,
+                        description: $('<div/>').html(DDG.strip_html(item.description)).text(),
```

Comment:
`item.description` has a lot of encoded html entities like `&quot;`. Passing only that text in `description` doesn't seem to be decoded when displaying the tiles. I also don't like this line. Any saner way of doing it?